apis/field_error.go

```diff
@@ -235,27 +237,26 @@ func containsString(slice []string, s string) bool {
 // FiledErrors. FieldErrors have their Paths combined (and de-duped) if their
```

Comment:
while here, FiledErrors -> FieldErrors?

Reply:
Eh, didn't notice this one.
Unforts, the PR automerged, so I'll send a new one.